code/game/objects/items/weapons/storage/uplink_kits.dm

```diff
@@ -178,10 +172,8 @@
 	name = "Freedom Implant (with injector)"
 
 /obj/item/weapon/storage/box/syndie_kit/imp_freedom/New()
+	new /obj/item/weapon/implanter/freedom(src)
```

Comment:
move this to be called after the supercall

Reply:
wait why?

Reply:
cause the code you were replacing was called after it. might be not important but definitely cleaner

Reply:
what about the other implant boxes?